<?xml version="1.0"?>
<launch>
	
	<arg name="tag" default="FL" />
	<node name="controller_node" pkg="ardrone_tutorials" type="joystick_controller2.py" output="screen" required="true">
		<!-- Configures the joystick button mapping -->
		<param name="ButtonEmergency"	value="0" />
		<param name="ButtonLand"		value="2" />
		<param name="ButtonTakeoff"		value="1" />
		<param name="ButtonAuto"		value="3" />
		

		<!-- Configures the joystick axis mapping -->
		<param name="AxisRoll"			value="0" />
		<param name="AxisPitch"			value="1" />
		<param name="AxisYaw"			value="2" />
		<param name="AxisZ"				value="5" />

		<!-- Configure Rotation D PAD -->
		<param name="AxisUD"			value="7" />
		<param name="AxisLR"			value="6" />

		<!-- Configures the joystick mapping -->
		<param name="ScaleRoll"			value="1" />
		<param name="ScalePitch"		value="1" />
		<param name="ScaleYaw"			value="1" />
		<param name="ScaleZ"			value="1" />
		
		<!-- <param name="Group"				value="$(arg Group)" /> -->
		<param name="DroneID"			value="$(arg tag)" />
		
	</node>


	<node name="joy_node" pkg="joy" type="joy_node" output="screen" clear_params="true">
		<param name="dev" type="str" value="/dev/input/js2"/>
	</node>

</launch>
